lib/honeybadger-api/fault.rb

```diff
@@ -16,7 +17,8 @@ def initialize(opts)
         @klass = opts[:klass]
         @action = opts[:action]
         @component = opts[:component]
-        @message = opts[:message]
+        @message = opts[:message],
+        @assignee = opts[:assignee],
```

Comment:
@ajish I haven't had a chance to look at the returned JSON from honeybadger but what does the `assignee` JSON look like? Is it something like:

```json
  "assignee": {
    "email": "westley@honeybadger.io", 
    "name": "Westley"
  }
```

If so then you will want to build out a new `User` object. We do this in other parts of the gem. Better to keep this consistent with the rest of the gem in my opinion. :+1: